Cargo.toml

```diff
@@ -33,11 +33,11 @@ async-graphql-axum = "5.0.10"
 async-lock = "2.8.0"
 async-stream = "0.3.5"
 async-trait = "0.1.74"
-aws-config = "0.14.0"
-aws-sdk-dynamodb = "0.14.0"
-aws-sdk-s3 = "0.14.0"
-aws-smithy-http = "0.44.0"
-aws-types = "0.14.0"
+aws-config = "0.56.1"
+aws-sdk-dynamodb = "0.34.0"
```

Comment:
I believe `0.35.0` is the last version: https://crates.io/crates/aws-sdk-dynamodb/versions

Reply:
Ok, it was after I started working on the problem.

Reply:
And that turns out to be quite as hard. Will try.

Reply:
Done.